Signum.React/Scripts/Operations/CellOperation.tsx

```diff
@@ -0,0 +1,203 @@
+  }
+}
+
+CellOperationButton.getText = (icoc: CellOperationContext): React.ReactNode => {
+
+  if (icoc.settings && icoc.settings.text)
+    return icoc.settings.text(icoc);
+
+  return <>{CellOperationButton.simplifyName(icoc.operationInfo.niceName)}{icoc.operationInfo.canBeModified ? <small className="ms-2">{OperationMessage.MultiSetter.niceToString()}</small> : null}</>;
+
+};
+
+CellOperationButton.simplifyName = (niceName: string) => {
+  const array = new RegExp(OperationMessage.CreateFromRegex.niceToString()).exec(niceName);
+  return array ? (niceName.tryBefore(array[1]) ?? "") + array[1].firstUpper() : niceName;
+}
+
+function confirmInNecessary(coc: CellOperationContext): Promise<boolean> {
+
+  const confirmMessage = getConfirmMessage(coc);
+
+  if (confirmMessage == undefined)
+    return Promise.resolve(true);
+
+  return MessageModal.show({
+    title: OperationMessage.Confirm.niceToString(),
+    message: confirmMessage,
+    buttons: "yes_no",
+    icon: "warning",
+    style: "warning",
+  }).then(result => { return result == "yes"; });
+}
+
+function getConfirmMessage(coc: CellOperationContext) {
+  if (coc.settings && coc.settings.confirmMessage === null)
+    return undefined;
+
+  if (coc.settings && coc.settings.confirmMessage != undefined)
+    return coc.settings.confirmMessage(coc);
+
+  if (coc.operationInfo.operationType == "Delete") {
+    const lite = coc.lite;
+    if (lite) {
+      return OperationMessage.PleaseConfirmYouWouldLikeToDelete0FromTheSystem.niceToString().formatHtml(<strong>{lite.toStr} ({getTypeInfo(lite.EntityType).niceName} {lite.id})</strong>);;
+    }
+  }
+
+  return undefined;
+}
+
+
+export function defaultCellOperationClick(coc: CellOperationContext, ...args: any[]): Promise<void> {
+
+  coc.event!.persist();
+
+  return confirmInNecessary(coc).then(conf => {
+    if (!conf)
+      return;
+
+    switch (coc.operationInfo.operationType) {
+      case "ConstructorFromMany":
+        throw new Error("ConstructorFromMany operation can not be in column");
+
+      case "ConstructorFrom":
+        if (coc.lite) {
+          return API.constructFromLite(coc.lite, coc.operationInfo.key, ...args)
+            .then(coc.onConstructFromSuccess ?? (pack => {
+              if (pack?.entity.id != null)
+                Navigator.raiseEntityChanged(pack.entity);
+              notifySuccess();
+              return Navigator.createNavigateOrTab(pack, coc.event!)
+                .then(() => { })
+            }));
+        };
+      case "Execute":
+        return API.executeLite(coc.lite, coc.operationInfo.key, ...args)
+          .then(coc.onExecuteSuccess ?? (pack => {
+            coc.cellContext.refresh?.();
+            coc.raiseEntityChanged();
+            notifySuccess();
+          }));
+      case "Delete":
+        return API.deleteLite(coc.lite, coc.operationInfo.key, ...args)
+          .then(coc.onDeleteSuccess ?? (() => {
+            coc.raiseEntityChanged();

```

Comment:
No cellContext?.refresh() ??

Reply:
Yes definitely, it's missed there
added